project/settings.sc

```diff
@@ -778,13 +778,13 @@ trait FormatNativeImageConf extends JavaModule {
 }
 
 trait ScalaCliScalafixModule extends ScalafixModule {
+
+  override def semanticDbVersion = T.input(Deps.Versions.scalaMeta)
```

Comment:
I don't really get why this needs to be an `input` making it re-evaluate every time. In my own plugins I was able to just make this a string, but I'm also on `0.11.x`. So no idea if there is a way around this or not.

Reply:
@ckipp01, there's one difference  to note: `scala-cli` is still using Mill in version `0.10.12`.

Reply:
> @ckipp01, there's one difference to note: `scala-cli` is still using Mill in version `0.10.12`.

Yup, that's what I meant by mentioning my usage of 0.11.x. Is there any plans to update? I'm assuming with all the breaking changes that it won't be a small task to update this build since it's so large.

Reply:
On Monday mill should be updated to `0.11.x`.

Reply:
The upgrade to Mill version `0.11.1` is currently in progress - https://github.com/VirtusLab/scala-cli/pull/2297. 

It should be merged in the coming days.

Reply:
https://github.com/VirtusLab/scala-cli/pull/2297 was merged. So ScalaCLI uses now Mill `0.11.1` .

Reply:
Perfect, I just rebased.